app/Repositories/PhotoRepository.php

```diff
@@ -76,6 +76,72 @@ public function create(array $data, $signupId)
     }
 
     /**
+     * Update an existing photo post and signup.
+     *
+     * @param \Rogue\Models\Photo $signup
+     * @param array $data
+     *
+     * @return \Rogue\Models\Photo
+     */
+    public function update($signup, $data)
+    {
+        // Update the signup's quantity and why_participated data.
+        // We will always update these since we can't tell if this has been changed in a good way yet.
+        $data['quantity_pending'] = $data['quantity'];
+        $signup->fill(array_only($data, ['quantity_pending', 'why_participated']));
+        $signup->save();
+
+        // If there is a file, create a new photo post.
+        if (isset($data['file'])) {
+            return $this->create($data, $signup->id);
+        } else {
+            // If it doesn't add a new photo, a new event won't be created. Create the event here for an updated signup.
+            // @TODO: right now, an updated signup's event_type is recorded as post_photo. Depending on future decisions, we might want to update this.
+            Event::create($data);
+        }
+
+        return $signup;
+    }
+
+    /**
+     * Updates a photo(s)'s status after being reviewed.
+     *
+     * @param array $data
+     *
+     * @return
+     */
+    public function reviews($data)
+    {
+        $reviewedPhotos = [];
+
+        foreach ($data as $review) {
+            if ($review['rogue_event_id'] && ! empty($review['rogue_event_id'])) {
```

Comment:
should this be an `isset()` check? Otherwise it looks like both of these statements are checking if it is false. so like `if (isset($review['rogue_event_id']) && ! empty($review['rogue_event_id']))`